experimental/dds/tree2/src/feature-libraries/editable-tree/editableTreeBinder.ts

```diff
@@ -102,7 +113,7 @@ export interface DataBinder<B extends OperationBinderEvents | InvalidationBinder
 	register<K extends keyof Events<B>>(
 		anchor: EditableTree,
 		eventType: K,
-		eventTrees: BindTree[],
+		eventTrees: BindPolicy[],
```

Comment:
nit: update `eventTrees` doc comment to say `BindPolicy` rather than `BindTree`.

Reply:
Done.